chainerx/__init__.py

```diff
@@ -15,6 +15,12 @@
 
 if _available:
     from numpy import dtype  # NOQA
+    from numpy import (  # NOQA
+        Inf, Infinity, NAN, NINF, NZERO, NaN, PINF, PZERO,
+        e, euler_gamma,
+        inf, infty, nan,
+        newaxis,
```

Comment:
`newaxis` is already defined in the python binding in `python/array_index.cc`. How about removing the binding code?

Reply:
Thanks, fixed as suggested.